packages/akar/src/a-popper/a-popper-content.vue

```diff
@@ -0,0 +1,370 @@
+    }),
+    props.prioritizePosition
+    && props.avoidCollisions
+    && flip({
+      ...detectOverflowOptions.value,
+    }),
+    props.avoidCollisions
+    && shift({
+      crossAxis: !!props.prioritizePosition,
+      limiter: props.sticky === 'partial' ? limitShift() : undefined,
+      mainAxis: true,
+      ...detectOverflowOptions.value,
+    }),
+    !props.prioritizePosition
+    && props.avoidCollisions
+    && flip({
+      ...detectOverflowOptions.value,
+    }),
+    size({
+      ...detectOverflowOptions.value,
+      apply: ({ availableHeight, availableWidth, elements, rects }) => {
+        const { height: anchorHeight, width: anchorWidth } = rects.reference;
+        const contentStyle = elements.floating.style;
+        contentStyle.setProperty(
+          '--akar-popper-available-width',
+          `${availableWidth}px`,
+        );
+        contentStyle.setProperty(
+          '--akar-popper-available-height',
+          `${availableHeight}px`,
+        );
+        contentStyle.setProperty(
+          '--akar-popper-anchor-width',
+          `${anchorWidth}px`,
+        );
+        contentStyle.setProperty(
+          '--akar-popper-anchor-height',
+          `${anchorHeight}px`,
+        );
+      },
+    }),
+    arrow.value
+    && floatingUIarrow({ element: arrow.value, padding: props.arrowPadding }),
+    transformOrigin({
+      arrowHeight: arrowHeight.value,
+      arrowWidth: arrowWidth.value,
+    }),
+    props.hideWhenDetached
+    && hide({ strategy: 'referenceHidden', ...detectOverflowOptions.value }),
+  ] as Array<Middleware>;
+});
+
+const { floatingStyles, isPositioned, middlewareData, placement } = useFloating(
+  rootContext.anchor,
+  floatingRef,
+  {
+    middleware: computedMiddleware,
+    placement: desiredPlacement,
+    strategy: 'fixed',
+    whileElementsMounted: (...args) => {
+      return autoUpdate(...args, {
+        animationFrame: props.updatePositionStrategy === 'always',
+      });
+    },
+  },
+);
+
+const placedSide = computed(
+  () => getSideAndAlignFromPlacement(placement.value)[0],
+);
+const placedAlign = computed(
+  () => getSideAndAlignFromPlacement(placement.value)[1],
+);
+
+watchEffect(() => {
+  if (isPositioned.value) {
+    emits('placed');
+  }
+});
+
+const cannotCenterArrow = computed(
+  () => middlewareData.value.arrow?.centerOffset !== 0,
+);
+
+const contentZIndex = ref('');
+watchEffect(() => {
+  if (contentElement.value) {
+    contentZIndex.value = window.getComputedStyle(contentElement.value).zIndex;
+  }
+});
+
+const arrowX = computed(() => middlewareData.value.arrow?.x ?? 0);
+const arrowY = computed(() => middlewareData.value.arrow?.y ?? 0);
+
+providePopperContentContext({
+  arrowX,
+  arrowY,
+  onArrowChange: (element) => {
+    arrow.value = element;
+  },
+  placedSide,
+  shouldHideArrow: cannotCenterArrow,
+});
+</script>
+
+<template>
+  <div
+    ref="floatingRef"
+    data-akar-popper-content-wrapper=""
+    :style="{
```

Comment:
Can we move this into the script section

Reply:
I've modified the code. 

Kindly inform me if it's not enough, otherwise, please resolve the conversation.